src/screens/home/components/benefits.js

```diff
@@ -89,83 +43,85 @@ class Benefits extends React.Component {
   render() {
     const styles = this.getStyles();
     return (
-      <div style={{ padding: `${VictorySettings.gutter}px 0` }}>
+      <div className="Benefits">
 
-        <div style={styles.flex}>
-          <div style={styles.flexTextItem}>
-            <h3>
-              Robust
-            </h3>
-            <p>
-              Area charts. Scatter plots. Voronoi polygons. Easy-to-use components for complex charting.
-            </p>
-          </div>
-          <div style={styles.flexTextItem}>
-            <h3>
-              Flexible
-            </h3>
-            <p>
-              Fully-contained, reusable data visualization elements are responsible for their own styles and behaviors.
-            </p>
-          </div>
-          <div style={styles.flexTextItem}>
-            <h3>
-              Native
-            </h3>
-            <p>
-              Extend the Victory experience on Android and iOS platforms with an identical&nbsp;API. <br/>
-              npm install victory-native
-            </p>
+        <div className="Container">
+          <div className="Grid Grid--gutters Grid--full medium-Grid--1of3">
+            <div className="Grid-cell u-textCenter">
+              <h3>
+                Robust
+              </h3>
+              <p>
+                Area charts. Scatter plots. Voronoi polygons. Easy-to-use components for complex charting.
+              </p>
+            </div>
+            <div className="Grid-cell u-textCenter">
+              <h3>
+                Flexible
+              </h3>
+              <p>
+                Fully-contained, reusable data visualization elements are responsible for their own styles and behaviors.
+              </p>
+            </div>
+            <div className="Grid-cell u-textCenter">
+              <h3>
+                Native
+              </h3>
+              <p>
+                Extend the Victory experience on Android and iOS platforms with an identical&nbsp;API. <br/>
+                <code>npm install victory-native</code>
+              </p>
+            </div>
           </div>
-        </div>
 
-        <div
-          style={{
-            display: "block",
-            fontFamily: VictorySettings.sansSerif,
-            marginTop: `${VictorySettings.gutter}px`,
-            marginRight: "3vw",
-            marginLeft: "3vw",
-            textAlign: "center"
-          }}
-        >
-          <RadiumLink style={styles.link} to="/docs">
-            Get Started <Icon glyph="internal-link" />
-          </RadiumLink>
+          <div className="Benefits-btn">
+            <RadiumLink style={styles.link} to="/docs">
```

Comment:
Agreed! This particular link has a background image with an `svg`, which requires an update to the archetype with the post-css svg support you mentioned, which I haven't gotten around to doing. So, I left it in a 'working state' for now... 

There is still some Radium tucked away in certain components. Ideally, it'll all be ripped out, but the tight deadline meant I couldn't get to everything at once. :(